.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+me: Bug Report
+about: If things aren't working as expected.
+
+---
+
+## Bug Report
+
+<!-- 
+Thanks for filing an issue! Before hitting the button, please answer these questions.
+
+Fill in as much of the template below as you can. If you leave out information, we can't help you as well.
+-->
+
+**What did you do?**
+A clear and concise description of the steps you took (or insert a code snippet).
+
+**What did you expect to see?**
+A clear and concise description of what you expected to happen (or insert a code snippet).
+
+**What did you see instead? Under which circumstances?**
+A clear and concise description of what you expected to happen (or insert a code snippet).
+
+
+**Environment**
+* operator-sdk version:
```

Comment:
We may need to have two version here (maybe?) 1. is the binary's version and 2. the version from Gopkg.Toml.

Reply:
Gopkg.toml exists only for `go` type operators, right? Might need to clarify that that version only applies to `go` operators.

Reply:
There could also be a potential mismatch of the version from the operator-sdk binary and the base image for the ansible operator that a user is using? Don't know how to capture that though.

Reply:
@shawn-hurley The ansible-operator logs should show the version of the vendored sdk.
```
INFO[0000] Go Version: go1.10.3
INFO[0000] Go OS/Arch: linux/amd64
INFO[0000] operator-sdk Version: 0.0.6+git
INFO[0000] Starting to serve on 127.0.0.1:8888
``` 
We could ask them to check the Gopkg.toml for a Go type operator. And the logs of their operator for an ansible type operator to check the version of the sdk. Assuming they can run the operator.